tests/SeleniumTests/__init__.py

```diff
@@ -68,8 +68,10 @@ def tearDown(self):
 
 class ChromeTests(SeleniumGenericTests):
     def setUp(self):
+        options=ChromeOptions()
+        options.add_argument('--no-sandbox')
```

Comment:
> such a configuration is unsupported and highly discouraged

https://chromedriver.chromium.org/help/chrome-doesn-t-start

Reply:
Thanks for your review. This change in test has been reverted, since there is a fix from #2017